### PR TITLE
[CHK-1348]add optional fields to cart request

### PR DESCRIPTION
**Pull request by ciuffagianluca**
Add `idCart` and `allCCP` fields to cart request

### List of changes

Add these fields to the body of the request. Both are optional

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

File: src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl [hunk 1]
```diff
@@ -1448,6 +1448,14 @@
                 "example": "www.comune.di.prova.it/pagopa/error.html"
               }
             }
+          },
```
Comment (pietro-tota) [inline src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl:1451]: Are we still updating this api group? As I know this one is marked for removal waiting for frontend upgrade in UAT, so I will not upgrade also this API group upgrading only the one into ecommerce-app domain.
What do you think?

**Review by pietro-tota — APPROVED** [review]
Just a comment

File: src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl (hunk 1 above)
Comment (ciuffagianluca) [reply src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl:1451]: You're right. I updated to be 100% compliant

Comment (ciuffagianluca) [reply src/core/api/checkout/checkout_ecommerce/v1/_openapi.json.tpl:1451]: Removed

**Review by pietro-tota — APPROVED** [review]

File: src/core/api/checkout/checkout_ec/v1/_openapi.json.tpl [hunk 2]
```diff
@@ -159,6 +159,14 @@
                 "example": "www.comune.di.prova.it/pagopa/error.html"
               }
             }
+          },
```
Comment (infantesimone) [inline src/core/api/checkout/checkout_ec/v1/_openapi.json.tpl:162]: to delete

**Review by infantesimone — CHANGES_REQUESTED** [review]

**Review by infantesimone — APPROVED** [review]

**Pull request merged**